src/main/java/org/mineacademy/fo/command/SimpleCommand.java

```diff
@@ -1144,10 +1144,10 @@ protected String replacePlaceholders(String message) {
 	 * @param message
 	 * @return
 	 */
-	private String replaceBasicPlaceholders0(String message) {
+	private String replaceBasicPlaceholders0(final String message) {
 
 		// Replace hard variables
-		message = Variables.replace(message, null);

```

Comment:
Thanks, although I am not sure if the messages variable gets altered if you do not assign it. I am missing something?